### PR TITLE
Document that bash is required for the install script

### PR DESCRIPTION
**Pull request by danmoseley**
Unless and until https://github.com/dotnet/install-scripts/issues/173 is fixed.

cc @mthalman @bozturkMSFT

### CODE REVIEW

File: docs/core/install/linux-scripted-manual.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ For .NET apps that use the *System.Drawing.Common* assembly, you'll also need th
 
 The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the **SDK** and **Runtime**. You can download the script from <https://dot.net/v1/dotnet-install.sh>.
 
+The script currently requires bash. On some distros this needs to be installed. On Alpine, for example, you would install bash with:
+
+```bash
+apk add bash
+```
+
```
Comment (adegeo) [inline docs/core/install/linux-scripted-manual.md:85]: For simplicity, I would just make this generic.

```suggestion
>![IMPORTANT]
>Bash is required to run the script.
```

Comment (danmoseley) [reply docs/core/install/linux-scripted-manual.md:85]: Note that in the distro specific topics we helpfully share the command to use, eg https://docs.microsoft.com/en-us/dotnet/core/install/linux-alpine#dependencies

But I think we can assume folks know (or can figure out) how to install it.

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
@danmoseley Are you happy with me merging this without the other's review?

**Review by bekir-ozturk — APPROVED** [review]
Thanks a lot @danmoseley. You are always one step ahead of us 🙂

**Comment by danmoseley** [comment]
great @adegeo I think we can merge.

**Review by mthalman — APPROVED** [review]

**Pull request merged**